lib/tasks/browserstack.rake

```diff
@@ -24,9 +24,9 @@ namespace :browserstack do
       # Fire up a tunnel in the background and wait until it is ready.
       r, w = IO.pipe
       pid = spawn("java -jar vendor/BrowserStackTunnel.jar -skipCheck #{ENV['BS_PASSWORD']} localhost,3000,0", out: w)
-      while content = r.readline
-        break if content == "Press Ctrl-C to exit\n"
-      end
+      content = r.readline
+
+      break if content == "Press Ctrl-C to exit\n" while content
```

Comment:
I don't think this works the same: in the old form, `r.readline` is evaluated on every loop.